manifests/params.pp

```diff
@@ -7,6 +7,36 @@
   $epel_includepkgs = absent
   $epel_exclude = absent
 
+  # ELREPO Settings.
+  # ELREPO has four repos, "main", "testing", "kernel" and "extras".
+  $elrepo_main_url = "http://elrepo.org/linux/elrepo/el${::operatingsystemmajrelease}/${::architecture}"
+  $elrepo_main_mirrors = "http://elrepo.org/mirrors-elrepo.el${::operatingsystemmajrelease}"
+  $elrepo_main_enabled = '1'
+  $elrepo_main_gpgcheck = '1'
+  $elrepo_main_includepkgs = absent
+  $elrepo_main_exclude = absent
+
+  $elrepo_testing_url = "http://elrepo.org/linux/testing/el${::operatingsystemmajrelease}/${::architecture}}"
+  $elrepo_testing_mirrors = "http://elrepo.org/mirrors-elrepo.el${::operatingsystemmajrelease}"
+  $elrepo_testing_enabled = '1'
```

Comment:
By default ElRepo only enables the main repo and leaves testing, kernel, and extras disabled by default. We should do the same here.